### ISSUE TITLE
Add New Features

### ISSUE THREAD

**Issue by acmjec** [acmjec/Hacktoberfest2023-Android_7_Min_Workout#7]
Implement new features to this app to make it better.

**Comment by TusharB4321** [comment]
**Before:**  When you click start button it automatically start exercise and there is no rest time and interval time for doing exercise for the user and finish all exercise in less than 10s.

**After:**  I am adding new feature when you click start button it take 9s countdown and then start first exercise  and after 30s interval next instruction is occured and it take time 9s so user can rest 9s and after next exercise start and so on    

so if you like this feature. So give me this opportunity so i can add my PR.

**Comment by Maan-Sharma** [comment]
Its  Sounds Great !! Lets see what mentors think about it. @TusharB4321

**Comment by acmjec** [comment]
Yes @TusharB4321 , You can add 10 secs timer before first Exercise. And for testing purpose, i have set the rest and exercise timer for 1 second. I am assigning this issue to you. Start Contributing. Happy Hacking!!

**Comment by TusharB4321** [comment]
@acmjec thanks for assigning and yes, I am thinking of new features to add
